FirebaseRemoteConfigSwift/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 9.7.0
```

Comment:
Hmm, I thought 9.6.0 was our last 9.X release. cc: @paulb777

Reply:
Updated to 10.0